packages/frontend/src/views/Login.vue

```diff
@@ -48,8 +48,14 @@
               </form>
             </div>
             <p class="has-text-link">
-              <a class="has-text-link" href="https://github.com/tjarbo/discord-moodle-bot/">Github Repository</a> &nbsp;·&nbsp;
-              <a class="has-text-link" href="https://github.com/tjarbo/discord-moodle-bot/wiki">Brauchst du Hilfe?</a>
+              <a
+                class="has-text-link"
+                href="https://github.com/tjarbo/discord-moodle-bot/"
+              >Github Repository</a> &nbsp;·&nbsp;
```

Comment:
Was ist das für ein Zeichen dazwischen?
```suggestion
              >Github Repository</a> &nbsp;&nbsp;
```

Reply:
Das ist ein optischer Trenner - jap der ist auf der Oberfläche sichtbar :smile: